spec/controllers/service_controller_spec.rb

```diff
@@ -31,6 +31,18 @@
         ['The Greatest Service in the World', nil],
       ])
     end
+
+    it 'sets the breadcrumbs without a delivery organisation' do
+      allow(service).to receive(:delivery_organisation) { nil }
+
+      get :show, params: { id: '2' }
+
+      expect(page.breadcrumbs.map {|crumb| [crumb.name, crumb.url] }).to eq([
```

Comment:
Just to save some pain later, when the linter is merged this line ```{|crumb| ...}``` will complain because of a missing space between { and |.  This seems to be the most common error it catches.